arch/arm64/boot/dts/qcom/sdm636-xiaomi-tulip.dts

```diff
@@ -195,6 +195,20 @@
 
 &blsp_i2c1 {
 	status = "okay";
+
+	touchscreen: novatek@62 {
```

Comment:
The node should be named "touchscreen" and label is not needed (you don't reference `&touchscreen` anywhere, do you?)

https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/tree/Documentation/devicetree/bindings/writing-bindings.rst#n24

> use node names matching the class of the device

Reply:
```diff
-touchscreen: novatek@62 {
+touchscreen@62 {
```